GNUmakefile

```diff
@@ -237,11 +237,10 @@ objs: $(OBJS)
 ###############
 
 # developer.android.com/ndk/downloads -> ~/android-ndk-xxx
-
-ANDROID_NDK = $(HOME)/android-ndk-r25
+# set an environment variable pointing ANDROID_NDK_ROOT to ~/android-ndk-xxx
```

Comment:
```suggestion
# override the ANDROID_NDK_ROOT environment variable to target your NDK install path

ifndef ANDROID_NDK_ROOT
ANDROID_NDK_ROOT = $(HOME)/android-ndk-r25
endif
```

What about still having a default path? This would both:
* give an hint on which version of the NDK libmatoya is supposed to be built against
* avoid requiring an extra step for developers who don't want to globally alter their environment